source/renderer/app/components/status/DaedalusDiagnostics.js

```diff
@@ -563,6 +613,37 @@ export default class DaedalusDiagnostics extends Component<Props, State> {
                   />
                 )
               )}
+              {getRow(
+                'meetsHardwareRequirementsLabel',
+                <PopOver
+                  content={intl.formatMessage(
+                    meetsHardwareRequirements
```

Comment:
if we only check this variable at startup we could memoise this who section as it is being checked so many times

Reply:
I'll submit it in a separate PR if that's ok? Adding it to the todo list

Reply:
ok no worries